examples/todomvc/test/actions/todos.spec.js

```diff
@@ -1,46 +1,52 @@
 import expect from 'expect'
-import * as types from '../../constants/ActionTypes'
 import * as actions from '../../actions'
 
 describe('todo actions', () => {
   it('addTodo should create ADD_TODO action', () => {
     expect(actions.addTodo('Use Redux')).toEqual({
-      type: types.ADD_TODO,
+      type: 'ADD_TODO',
       text: 'Use Redux'
     })
   })
 
+  it('setVisibilityFilter should create SET_VISIBILITY_FILTER action', () => {
+    expect(actions.setVisibilityFilter('active')).toEqual({
+      type: 'SET_VISIBILITY_FILTER',
+      filter: 'active'
+    })
+  })
+
   it('deleteTodo should create DELETE_TODO action', () => {
     expect(actions.deleteTodo(1)).toEqual({
-      type: types.DELETE_TODO,
+      type: 'DELETE_TODO',
```

Comment:
I would like to keep action types constants in this example. `todos` is more simplistic and can avoid it, but this looks more like a real app so I think they are beneficial.